hack/make_metricsdoc.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+function get_metrics_url() {
+	pod_name=$1
+	pod_yaml=$(kubectl get pod -n kube-system $pod_name -o yaml)
+	host_ip=$(awk '/[[:space:]]hostIP:/{print $2}' <<< "$pod_yaml")
+	host_port=$(awk '/[[:space:]]hostPort:/{print $2}' <<< "$pod_yaml")
```

Comment:
kubectl supports:
```bash
kubectl get pod -n kube-system $pod_name --template={{.status.hostIP}}
```
which is nicer than using `awk` from a readability / maintainability perspective IMO. Same for `hostPort`.

Reply:
Will use -o jsonpath. Would like to use this for the token/cert retrieval, and --template doesn't handle ca.crt very well (dot in the key).
So I'll use the same flag for all kubectl calls.